### ISSUE TITLE
Substep does not work. Did I miss something?

### ISSUE THREAD

**Issue by yoosofan** [regebro/hovercraft#214]
Thanks @regebro, hovercraft developer, for your wonderful and easy tool.
I tried to use substep based on [presentations.html](https://hovercraft.readthedocs.io/en/latest/presentations.html#showing-lists-item-by-item) using Hovercraft! 2.6 on Ubuntu 18.04, but it did not work. What did I miss?
```
----

JSON Values
=================

.. class:: substep

    *   a string
    *   a number
    *   an object (JSON object)
    *   an array
    *   a boolean
    *   null

----


JSON Values
=================

.. class:: substep

    *   a string

        *   a number

        *   an object (JSON object)

        *   an array

        *   a boolean

        *   null

----

.. class:: substep

    *   a string

    *   a number

    *   an object (JSON object)

    *   an array

    *   a boolean

    *   null

----
```

**Comment by regebro** [comment]
Thanks for your kind words. Substeps was first implemented in version 2.7beta1, so they don't work in 2.6.

**Comment by yoosofan** [comment]
Thanks for your answer. After upgrading to latest release, it works very well. 
The following instructions help anyone who wants to upgrade to latest release (Thanks to [Casey](https://stackoverflow.com/a/43697920/886607)
```
curl -L https://api.github.com/repos/regebro/hovercraft/tarball > hovercraft.tar.gz
tar -xvf hovercraft.tar.gz
```
Go to the folder by the name something like this
```
regebro-hovercraft-
```
Use tab key for the rest (cd regebro-hovercraf  )
Install it
```
 pip --user install -e ./
```
